src/test/kotlin/com/igorwojda/string/longestword/challenge.kt

```diff
@@ -4,7 +4,9 @@ import org.amshove.kluent.shouldBeEqualTo
 import org.junit.jupiter.api.Test
 
 private fun longestWord(str: String): String {
-    TODO("not implemented")
+    val vectorPentruStr = str.replace(Regex("[^A-Za-z0-9 ]")," ")
```

Comment:
nitpick: chains of functions calls should be formatted on multiple lines for better readibility